flask_stormpath/models.py

```diff
@@ -73,14 +74,24 @@ def delete(self):
 
     def to_json(self):
         writable_attrs = (
+            'href',
+            'modified_at',
+            'created_at',
+            'status',
             'username',
             'email',
             'given_name',
             'middle_name',
             'surname',
-            'status')
-        return json.dumps({'account': {
-            key: getattr(self, key, None) for key in writable_attrs}})
+            'full_name'
+        )
+
+        json_data = {'account': {}}
+        for key in writable_attrs:
+            attr = getattr(self, key, None)
+            json_data['account'][key] = (
+                attr if not isinstance(attr, datetime) else attr.isoformat())
+        return json.dumps(json_data)
```

Comment:
http://stackoverflow.com/questions/23285558/datetime-date2014-4-25-is-not-json-serializable-in-django